libs/ngx-dsfr/alert/src/lib/testing/alert.component.spec.ts

```diff
@@ -115,9 +116,12 @@ describe('DsfrAlertComponent', () => {
 
```

Comment:
The `it('shouldn\'t have a description'` unit test doesn't pass.
Since we don't remove the `<p>` tag anymore, `description` is an empty string rather than `undefined`